controllers/managedocs_controller.go

```diff
@@ -63,6 +114,7 @@ func (r *ManagedOCSReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	log := r.Log.WithValues("req.Namespace", req.Namespace, "req.Name", req.Name)
 	log.Info("Reconciling ManagedOCS")
 
+	var err error
```

Comment:
I would like to have an err var defined (every time) when needed and not globally, it allows us to track the right var when watching dumps (each instance will have a diff memory address)

Reply:
done